packages/cw-orch-networks/src/chain_info.rs

```diff
@@ -13,7 +13,11 @@ use cw_orch_core::{
 impl Into<RegistryChainInfo> for ChainInfo<'_> {
     fn into(self) -> RegistryChainInfo {
         RegistryChainInfo {
-            chain_name: self.network_info.id.to_string(),
+            chain_name: match self.kind {
+                ChainKind::Local => self.chain_name.to_string(),
+                ChainKind::Mainnet => self.chain_name.to_string(),
+                ChainKind::Testnet => format!("testnets/{}", self.chain_name),
+            },
```

Comment:
```suggestion
            chain_name: self.chain_name,
```
I would prefer to just return it as is, IMO we should have as close result as possible to fetching this `chain_name`. User can add path themself based of `ChainKind`, if he needs to. 
![image](https://github.com/AbstractSDK/cw-orchestrator/assets/91957742/ef30b432-0ef1-4893-a57e-ec32d0b33c98)

Reply:
Agree with this, we shouldn't deflect from chain registry with this.